runtime/src/main/java/io/quarkiverse/semantickernel/SemanticKernelProducer.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkiverse.semantickernel;
+
+import jakarta.enterprise.inject.Produces;
+import jakarta.inject.Inject;
+
+import com.azure.ai.openai.OpenAIAsyncClient;
+import com.microsoft.semantickernel.Kernel;
+import com.microsoft.semantickernel.SKBuilders;
+
+public class SemanticKernelProducer {
+    // Just a >placeholder<, before the actual implementation, to enable the
+    // semantic function registration and test
+
+    @Inject
+    OpenAIAsyncClient client;
+
+    @Produces
```

Comment:
Could you create an integrations test that Injects the Kernel ? Without an @ApplicationScoped, I'm not sure this could work

Reply:
If you agree to open a dedicated issue for such feature, we can work on it properly